src/config.c

```diff
@@ -352,6 +352,8 @@ TOKEN *new_constant(const char *con) {
     !strcmp("HAS_VTARG_SWITCH", con)? HAS_VTARG_SWITCH:
     !strcmp("HAS_VTARG_ADJ", con)? HAS_VTARG_ADJ:
     !strcmp("HAS_VTARG_READ", con)? HAS_VTARG_READ:
+    !strcmp("HAS_FOSC_ADJ", con)? HAS_FOSC_ADJ:
```

Comment:
Maybe give the shining new `str_eq()` a bit of an airing in this section (and perhaps elsewhere in this file seeing as it's being touched anyway)?

Reply:
Good idea! I've replaced (!)strcmp with str_eq in the entire file